src/connectors/autocomplete/__tests__/connectAutocomplete-test.ts

```diff
@@ -22,7 +22,7 @@ describe('connectAutocomplete', () => {
     });
 
     const initialConfig = {};
-    const helper = algoliasearchHelper({} as Client, '', initialConfig);
+    const helper = algoliasearchHelper({} as SearchClient, '', initialConfig);
```

Comment:
Instead of casting we can pass an actual client with [`createSearchClient`](https://github.com/algolia/instantsearch.js/blob/1a6dc518763132eb3223c078cb38841bca83b602/test/mock/createSearchClient.ts#L9-L21). This will be more consistent with the real usage.

Reply:
@francoischalifour Indeed. Do you mind if we do those changes in another pull request that actually tackles that problem?

Reply:
Sure, if you prefer!